### PR TITLE
Update translation_pt.xml

### PR DESCRIPTION
**Pull request by Sneekavis**

### CODE REVIEW

File: translations/translation_pt.xml [hunk 1]
```diff
@@ -8,418 +8,417 @@
...
+    <text name="CP_global_setting_controllerHudSelected_title" text="Ecrã secundário de opções acionado pelo controlador de jogo"/>
+    <text name="CP_global_setting_controllerHudSelected_tooltip" text="Usa e navega no ecrã secundário de opções com o controlador de jogo."/>
+    <text name="CP_global_setting_showActionEventHelp_title" text="Alerta de eventos"/>
+    <text name="CP_global_setting_showActionEventHelp_tooltip" text="Mostra o texto de ajuda com um alerta."/>
+    <text name="CP_global_setting_subTitle_pathfinder" text="Configurações do encontrar caminhos"/>
+    <text name="CP_global_setting_maxDeltaAngleAtGoal_title" text="Ângulo diferente para o objectivo"/>
+    <text name="CP_global_setting_maxDeltaAngleAtGoal_tooltip" text="Diferença máxima autorizada do ângulo até ao objectivo, valores mais altos indica que vai mais rápido, mas com menos exacto"/>
+    <text name="CP_global_setting_deltaAngleRelaxFactor_title" text="Factor de tolerância"/>
+    <text name="CP_global_setting_deltaAngleRelaxFactor_tooltip" text="Factor para diminuir a precisão de encontrar o caminho com iterações mal sucedidas, valores maiores são mais rápidos"/>
```
Comment (Tensuko) [inline translations/translation_pt.xml:280]: Please revert those changes:
CP_global_setting_maxDeltaAngleAtGoal_title
CP_global_setting_maxDeltaAngleAtGoal_tooltip
CP_global_setting_deltaAngleRelaxFactor_title
CP_global_setting_deltaAngleRelaxFactor_tooltip

You did remove the Deg, wich leads into missing translation.

File: translations/translation_pt.xml [hunk 2]
```diff
@@ -659,8 +658,8 @@ The course editor can be used to edit courses and custom fields.
 The loaded courses will be saved/overritten automatically, when the editor is closed.
 
 1) Editing a custom field can be done with a selected field on the ai menu page.
-2) Editing a course can be done in the course manager. 
-   There you can edit a saved course, but be sure you cleared the currently loaded course from the vehicle.
+2) Editing a course can be done in the course manager.  
+      There you can edit a saved course, but be sure you cleared the currently loaded course from the vehicle.
```
Comment (Tensuko) [inline translations/translation_pt.xml:662]: This is not translated, but added spaces, pls revert this one too.

**Review by Tensuko — COMMENTED** [review]
Some requested changes

**Comment by Sneekavis** [comment]
I have tried to reverted, but i couldn't find the way, you can't reverted?

**Comment by Tensuko** [comment]
Ok I can fix that later.

**Comment by Sneekavis** [comment]
Thank you, the next time i ask first before trying.
________________________________

**Pull request merged**